src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs

```diff
@@ -267,45 +268,47 @@ public class OutputCacheFilter : FilterProvider, IActionFilter, IResultFilter {
 
         protected virtual bool RequestIsCacheable(ActionExecutingContext filterContext) {
 
+            var itemDescriptor = string.Format("{0} (Controller: {1}, Action: {2})", filterContext.RequestContext.HttpContext.Request.RawUrl, filterContext.ActionDescriptor.ControllerDescriptor.ControllerName, filterContext.ActionDescriptor.ActionName);

```

Comment:
We previously logged the cache key as an identifier, but that will always be null now, so `itemDescriptor` is designed to be a lightweight replacement for that.